server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -87,6 +87,7 @@ class Elasticsearch {
 
     private static final String POLICY_PATCH_PREFIX = "es.entitlements.policy.";
     private static final String SERVER_POLICY_PATCH_NAME = POLICY_PATCH_PREFIX + "server";
+    private static final String APM_AGENT_PACKAGE_NAME = "co.elastic.apm.agent";
```

Comment:
This is unfortunate.

Reply:
It could go inside `ScopeResolver` I guess? I think I made it a constructor arg because unit tests passed in a different value.